### DISCUSSION TITLE
[BUG] Scrolls to top on mobile

### DISCUSSION THREAD

**Discussion by IIPoliII** [squidfunk/mkdocs-material#2802] [Q&A]
I have an issue on mobile phones.
When you scroll down a page it automatically goes back to the top again.

You can see the issue here : https://wiki.polisystems.ch/English/

Simply take a phone with Chrome or Firefox and it scrolls again to the top, but not in PC mode on the phone.
I am using a Xiaomi MI 10T Pro, on Android 10, latest chrome and Firefox.

Any idea?

**Comment by squidfunk** [comment]
I checked and it doesn’t happen on iOS, so I cannot reproduce. However, you can check the official docs which just got a slight improvement for the back-to-top button - it’s now rendered at the top. The change will be released soon and might solve your problem on the way.

**Reply by IIPoliII** [reply]
Ah totally possible this button is the issue, do you have an PR of this change?
Thanks for helping out.

**Reply by squidfunk** [reply]
https://github.com/squidfunk/mkdocs-material/commit/9f01704a73f3b020a90880002c2de0e57669166f is the relevant commit.

**Reply by IIPoliII** [reply]
Sadly it doesn't fix it for me.
It's bounded to "    - header.autohide" in my features section. If I disable it works perfectly.

But well I don't need this option
